plugins/kosync.koplugin/main.lua

```diff
@@ -30,12 +44,15 @@ function KOSync:init()
     self.kosync_username = settings.username
     self.kosync_userkey = settings.userkey
     self.kosync_auto_sync = not (settings.auto_sync == false)
+    self.kosync_whisper_forward = settings.whisper_forward or SYNC_STRATEGY.DEFAULT_FORWARD
+    self.kosync_whisper_backward = settings.whisper_backward or SYNC_STRATEGY.DEFAULT_BACKWARD
     self.kosync_device_id = G_reader_settings:readSetting("device_id")
     --assert(self.kosync_device_id)
     self.ui:registerPostInitCallback(function()
```

Comment:
probably makes more sense to call this trunk of code in `onReaderReady` listener.